### PR TITLE
Updating gurobi page

### PR DESCRIPTION
**Pull request by tpfau**
This is an update to the gurobi page with current information.

Things might change soon if there is an update to the license.

### CODE REVIEW

File: triton/apps/gurobi.rst [hunk 1]
```diff
@@ -9,37 +9,80 @@ License
...
+   Unfortunately the python gurobi packages installed via pip and via conda come with
+   two distinct package names ``gurobi`` for the anaconda package and ``gurobipy`` for 
+   the pip package. Normally, we install the guobi package in the anaconda environment, 
+   but there are some anaconda modules which have the gurobipy package. So you might need
+   to select the correct package.
 
-You can create this license file with the following command on the login node::
+.. admonition:: License Files for older Anaconda modules
+   
+   Older anaconda modules might not have the GRB_LICENSE_FILE environment variable set
```
Comment (rkdarst) [inline triton/apps/gurobi.rst:30]:
```suggestion
   Older anaconda modules on Triton might not have the GRB_LICENSE_FILE environment variable set 
```

File: triton/apps/gurobi.rst [hunk 2]
```diff
@@ -9,37 +9,80 @@ License
 
 Aalto University has a site-wide floating license for Gurobi.
 
-You can create a ``gurobi.lic`` file in your home folder. The file should contain the following single line:
+.. admonition:: Important notes
+   
+   As of writing of this Guide, Aalto only has a valid license for Gurobi 9.X and older.
+   Therefore Gurobi 10 cannot be run on triton.
```
Comment (rkdarst) [inline triton/apps/gurobi.rst:15]:
```suggestion
   Therefore Gurobi 10 cannot be run on triton. 
```

**Review by rkdarst — APPROVED** [review]
Nice update!

Do we need to wait for @simo-tuomisto's update of the anaconda modules to merge this, or are we good to go?  Seems like it should be accurate with the updated module...

**Comment by simo-tuomisto** [comment]
I'll merge this now and will create a separate PR on my updates.

**Pull request merged**